feature/interface/singleton/tests/singleton_with_breakouts/metadata.textproto

```diff
@@ -0,0 +1,24 @@
+# proto-file: github.com/openconfig/featureprofiles/proto/metadata.proto
+# proto-message: Metadata
+
+uuid:  "bf9109a0-9117-40a4-8417-aecfe36820ea"
+plan_id:  "RT-8"
+description:  "Singleton with breakouts"
+testbed: TESTBED_DUT
```

Comment:
looks like there is an issue with "TESTBED_DUT"  option . I see that binding file with isolated ports are not getting picked up for test  unless we pass testbed file. so either we need to fix this in ondatra or use other testbed types that suitable for test. Basically testbed type  say  ports are optional but script is expecting some ports .

Reply:
Changed it to DUT_ATE4